util/info.c

```diff
@@ -53,6 +53,7 @@ static const struct option longopts[] = {
 	{"help", no_argument, NULL, 'h'},
 	{"node", required_argument, NULL, 'n'},
 	{"port", required_argument, NULL, 'P'},
+	{"source", no_argument, NULL, 's'},
```

Comment:
Fabtests require an argument for -s.  It would be best to align with those tests.

Reply:
Are you just suggesting I use a different letter? What would be best?

Reply:
I'm suggesting we align with fabtests.

In this case, -s would require an address parameter as input.  So, the user would specify -s <src_addr>.  For now, only one of -s or -n would be allowed.

For the future...

It's unfortunate that specifying the node value requires '-n', rather than it simply being a command line argument (like it is for fabtests), but that's fixable.  We can keep -n for compatibility, and allow an open argument for the dst_addr.

If only -s <src_addr> is given, that defaults to setting FI_SOURCE, similar to fabtests, with src_addr passed as the node parameter.  If only <dst_addr> (or -n <dst_addr>) is given, dst_addr is passed as the node parameter.  If both -s <src_addr> and <dst_addr> are given, then <dst_addr> is passed as the node, with src_addr set as the src_addr in the hints.

See ft_getinfo() (shared.c) and ft_read_addr_opts() for details on how I think this should work. 

Again, for now, I would just have the -s and -n options be exclusive.